components/header/podIndicator/index.jsx

```diff
@@ -19,55 +19,97 @@
  * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 
-import React, { useEffect, useState } from "react";
+import React, { useState } from "react";
 import clsx from "clsx";
 import { createStyles, makeStyles } from "@material-ui/styles";
 import { useRouter } from "next/router";
 import { useBem } from "@solid/lit-prism-patterns";
-import { foaf, vcard } from "rdf-namespaces";
-import { getStringNoLocale } from "@inrupt/solid-client";
+import Popover from "@material-ui/core/Popover";
+import { Form, Input } from "@inrupt/prism-react-components";
+import Skeleton from "@material-ui/lab/Skeleton";
 import usePodOwnerProfile from "../../../src/hooks/usePodOwnerProfile";
-
 import styles from "./styles";
+import { resourceHref } from "../../../src/navigator";
+import { normalizeContainerUrl } from "../../../src/stringHelpers";
 
 const useStyles = makeStyles((theme) => createStyles(styles(theme)));
 
+export const clickHandler = (setAnchorEl) => (event) =>
+  setAnchorEl(event.currentTarget);
+
+export const closeHandler = (setAnchorEl) => () => setAnchorEl(null);
+
+export const submitHandler = (handleClose, setUrl) => async (
+  event,
+  url,
+  router
+) => {
+  event.preventDefault();
+  const containerUrl = normalizeContainerUrl(url);
+  await router.replace("/resource/[iri]", resourceHref(containerUrl));
```

Comment:
This should be `push` so that I can go "back" in my browser history.

Reply:
Fixed